docs/cases/cases-index.asciidoc

```diff
@@ -1,5 +1,7 @@
 include::cases-overview.asciidoc[leveloffset=+1]
 
+include::cases-req.asciidoc[leveloffset=+1]
```

Comment:
I think it makes more sense to place it under Cases, and to have it on the same level as the two other pages (Open and manage cases and Configure case settings):

```suggestion
include::cases-req.asciidoc[leveloffset=+2]
```